src/components/30-organisms/footer/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.1.27
+
+- Fix: If you use React the height adapts automatically to the number of elements. #2142
```

Comment:
As written, this is too hard to understand  (what are elements?? what height?).

Suggestion: If you use React, the footer height automatically adapts to the number of child elements.